### ISSUE TITLE
Minimap / Journeymap native

### ISSUE THREAD

**Issue by EmeraldSnorlax** [ImpactDevelopment/ImpactClient#1247]
Add a minimap / journeymap native to impact
Radar just seems like a half-baked attempt

**Comment by MastaNub** [comment]
And waypoints!

**Comment by EmeraldSnorlax** [comment]
Waypoints compatible with baritone? 😩😩

**Comment by AstragoYT** [comment]
we need it

**Comment by Cerbiac** [comment]
Possibly Features:
- Terrain Visualization
- Cave Visualization (Probably renders for a different y-value down. IDK how minimaps do it.)
- Waypoints (Which is its own fleshed out feature.)
- Waypoint-Baritone Integration
- Chunk Borders

Otherwise, it works perfectly fine, and does just about everything I need it to: show entities & show baritone path.

**Comment by DanCuber** [comment]
To add to Cerbiac's suggestions, visualizing newChunks on the minimap aswell could be useful (of have an option to have them only render on the minimap)

**Comment by realkarmakun** [comment]
Slime Chunks logging with writing cords in file would be great for seed reverse engineering

**Comment by realkarmakun** [comment]
I just understood that you need seed for automatic slime chunks finding

**Comment by Thunder33345** [comment]
~~but not every server's seed is public~~
a helper would be useful to assist seed reverse bruteforce

**Comment by sapioit** [comment]
You see, to find the seed, you need to know at least 14 slime chunks.

And yes, I also want the fully-fledged minimap.

**Comment by S-B99** [comment]
No, you only need 3 dungeons. Finding 14 clime chunks is insane.

**Comment by aaronfranke** [comment]
I would love to have this feature in Impact. Specifically, it would be nice to have something like Journeymap where you can press a button to open a full screen map.

**Comment by Toiisha** [comment]
anyone want some head? af1#2305 gives it for free
